tasks/st1105_primary_server_port_check.rb

```diff
@@ -32,8 +32,12 @@ def port_test(dest, port)
   true
 end
 
+
 results_json = []
+deprecation = {"This task is deprecated and will be removed in a future release. Please see this module's README for more information."}
```

Comment:
this is better, it will work fine, but this is an array with one string element, a hash is a key pair, matching the formatting of the rest of the json return would look like this:

{
'Message' => 'deprecation message',
}

Matching the formatting and data type of the existing content is important in general, particularly with tasks as it makes it easier to query the return data